ios/FluentUI/Vnext/HUD/HUDTokens.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+import UIKit
+
+/// Design token set for the `ActivityIndicator` control.
+open class HeadsUpDisplayTokens: ControlTokens {
+
+    /// The color of the squared background of the Heads-up Display.
+    open var backgroundColor: DynamicColor { DynamicColor(light: ColorValue(r: 0.129, g: 0.129, b: 0.129, a: 0.9),
+                                                          dark: ColorValue(r: 0.188, g: 0.188, b: 0.188, a: 1)) }
+
+    /// The color of the contents presented by the Heads-up Display.
+    open var foregroundColor: DynamicColor { DynamicColor(light: ColorValue(0xFFFFFF),
```

Comment:
Nit: can use `globalTokens.neutralColors[.white]` here and below.

Reply:
Sure! Updated!
This will be reviewed soon as part of the fluent2 color update as well.